app/labor/cache_buster.rb

```diff
@@ -36,6 +36,21 @@ def self.nginx_enabled?
     ApplicationConfig["OPENRESTY_PROTOCOL"].present? && ApplicationConfig["OPENRESTY_DOMAIN"].present?
   end
 
+  def self.openresty_path
+    "#{ApplicationConfig['OPENRESTY_PROTOCOL']}#{ApplicationConfig['OPENRESTY_DOMAIN']}"
+  end
```

Comment:
**suggestion: ** Should we move this to [URL](https://github.com/forem/forem/blob/master/app/lib/url.rb) or another helper so we can use it elsewhere in the future?